oneview/resource_logical_interconnect_group.go

```diff
@@ -103,7 +102,7 @@ func resourceLogicalInterconnectGroup() *schema.Resource {
 							Required: true,
 						},
 						"logical_port_config": {
-							Type:     schema.TypeList,
+							Type:     schema.TypeSet,
```

Comment:
From OV UI we can not set desired speed for Ethernet network, what will happen if user set desired_speed = ""?

Reply:
It will take up the default value.